Server/Plugins/APIDump/main_APIDump.lua

```diff
@@ -1635,6 +1635,75 @@ end
 
 
 
+local function DumpLuaCheck(a_API)
+	LOG("Creating file .luacheckrc...")
+	local file = io.open(".luacheckrc", "w")
+	
+	file:write([[
+-- This file is the config file for the tool named Luacheck
+-- Documentation: http://luacheck.readthedocs.io/en/stable/index.html
+
+-- Ignore unused function and loop arguments
+unused_args = false
+
+-- Allow self defined globals
+allow_defined = true
+
+-- Ignore this functions
+ignore =
+{
+	"Initialize",  -- Plugin
+	"OnDisable",  -- Plugin
+	"RegisterPluginInfoCommands",  -- InfoReg.lua
+	"RegisterPluginInfoConsoleCommands",  -- InfoReg.lua
+	"g_PluginInfo",  -- Info.lua
+}
+
+-- Ignore files / directories
+exclude_files =
+{
+	"tests/" -- CuberitePluginChecker
+}
+
+-- All globals from cuberite (classes, enums, functions)
+globals =
+{
+]])
+	
+	-- Add all class names
```

Comment:
Stale comment - the code adds not only class names, but also global constants and functions. How about `Export all global symbols` ?